pkg/controller/querier/querier.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package querier
+
+import (
+	"os"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	serviceName  = "antrea"
+	podName      = "POD_NAME"
+	podNamespace = "POD_NAMESPACE"
+	nodeName     = "NODE_NAME"
+)
+
+// GetSelfPod gets current pod.
+func GetSelfPod() v1.ObjectReference {
```

Comment:
GetSelfPod() is ok because it is completely the same in Controller and Agent. But others like GetSelfNode() is a little different as Agent one takes a parameter. Since Golang doesn't have overloading, maybe change their name to GetSelfAgentNode() and GetSelfControllerNode()?

Reply:
Or let's add a parameter: isAgent into GetSelfNode()

Reply:
Updated